Solutions/ThreatXCloud/Playbooks/CustomConnector/ThreatXCustomConnector/azuredeploy.json

```diff
@@ -0,0 +1,845 @@
+                                "current_password": {
+                                  "type": "string",
+                                  "description": "current_password"
+                                },
+                                "password": {
+                                  "type": "string",
+                                  "description": "password"
+                                },
+                                "given_name": {
+                                  "type": "string",
+                                  "description": "given_name"
+                                },
+                                "family_name": {
+                                  "type": "string",
+                                  "description": "family_name"
+                                },
+                                "phone": {
+                                  "type": "string",
+                                  "description": "phone"
+                                },
+                                "super_admin": {
+                                  "type": "string",
+                                  "description": "super_admin"
+                                },
+                                "tx_admin": {
+                                  "type": "string",
+                                  "description": "tx_admin"
+                                },
+                                "tenant_admin": {
+                                  "type": "boolean",
+                                  "description": "tenant_admin"
+                                },
+                                "read_only": {
+                                  "type": "boolean",
+                                  "description": "read_only"
+                                }
+                              },
+                              "description": "user"
+                            },
+                            "customer_name": {
+                              "type": "string",
+                              "description": "customer_name"
+                            },
+                            "channel_name": {
+                              "type": "string",
+                              "description": "channel_name"
+                            },
+                            "features": {
+                              "type": "array",
+                              "items": {
+                                "type": "string"
+                              },
+                              "description": "features"
+                            }
+                          },
+                          "description": "Ok"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "/tx_api/v1/lists": {
+              "post": {
+                "summary": "Get-Post Blacklist_Blocklist_Whitelist",
```

Comment:
[Optional] Mixing dashes and underscore doesn't look good. Instead of "Get-Post Blacklist_Blocklist_Whitelist" you can simply use "Get-Post Blacklist Blocklist Whitelist".

Reply:
sure rahul , let me check on this